modmail/bot.py

```diff
@@ -19,27 +23,130 @@ class ModmailBot(commands.Bot):
     Has an aiohttp.ClientSession and a ModmailConfig instance.
     """
 
-    main_task: asyncio.Task
     logger: ModmailLogger = logging.getLogger(__name__)
 
     def __init__(self, **kwargs):
         self.config = CONFIG
+        self.start_time: t.Optional[arrow.Arrow] = None  # arrow.utcnow()
+        super().__init__(
+            **kwargs,
+        )
         self.http_session: t.Optional[ClientSession] = None
-        self.start_time = arrow.utcnow()
-        super().__init__(command_prefix=commands.when_mentioned_or(self.config.bot.prefix), **kwargs)
 
-    async def create_session(self) -> None:
-        """Create an aiohttp client session."""
-        self.http_session = ClientSession()
+    @classmethod
+    def create(cls, *args, **kwargs) -> "ModmailBot":
+        """
+        Create a ModmailBot instance.
 
-    async def start(self, *args, **kwargs) -> None:
+        This configures our instance with all of our custom options, without making __init__ unusable.
```

Comment:
We don't expect actual customization of the bot call. Providing the ability to is nice, but this is really just dodging a large `__init__` method and we're probably fine to just have a big one, especially when there's a lot of initialization, like here. Some of these should also be moved out of here, like intents

Reply:
our bot requires the member intent, so where would you suggest the intents go to?

Reply:
Globals